src/tools/clang.lua

```diff
@@ -282,7 +283,12 @@
 --
 
 	function clang.getlinks(cfg, systemonly, nogroups)
-		return gcc.getlinks(cfg, systemonly, nogroups)
+		-- For visual studio projects, the links should match the MSC convention
```

Comment:
This is wrong, it should only apply to Windows projects, not all projects. Furthermore, this change should be made in the VS module.

Reply:
I can easily move the change to the VS module.  Just so I go the agreed course of action, would you say:

* Changes made in the vs2019 module
* Changes only affect the clang and msc toolsets for windows projects